### DISCUSSION TITLE
Network Goerli doesn't exist using HardHat

### DISCUSSION THREAD

**Discussion by GratefulLion21** [NomicFoundation/hardhat#3493] [Troubleshooting]
<img width="535" alt="Screen Shot 2022-12-29 at 9 59 53 AM" src="https://user-images.githubusercontent.com/77343843/209971578-12dbd58c-3a1b-4f4a-85e0-50332055f694.png">

**Comment by AmitKochar1** [comment]
are you still having this issue? if yes, I can help you out. The private key should be in hexa decimals, missing chainId and other spelling mistakes. If you have not resolved this yet. I am happy to help out.
